### PR TITLE
New version 1.5

### PR DESCRIPTION
**Pull request by zoranzoki21**
* Composer tests: Add jakub-onderka/php-parallel-lint, jakub-onderka/php-console-hightlighter, mediawiki/mediawiki-codesniffer and mediawiki/minus-x
* NPM tests: Add grunt-banana-checker and grunt-eslint
* Add translations for Serbian
* Move translating messages from AutoSitemap.i18n.php into the json files

### CODE REVIEW

File: AutoSitemap_body.php [hunk 1]
```diff
@@ -46,207 +46,220 @@
...
-                                                                                            NS_HELP_TALK,
-                                                                                            NS_CATEGORY_TALK
-                                                                                         ];
-
-if (!isset($wgAutoSitemap["exclude_pages"]    )) $wgAutoSitemap["exclude_pages"]       = [];
-if (!isset($wgAutoSitemap["priority"]         )) $wgAutoSitemap["priority"]            = [];
-if (!isset($wgAutoSitemap["freq"]             )) $wgAutoSitemap["freq"]                = "daily";
-
-if (!isset($wgAutoSitemap["header"]           )) $wgAutoSitemap["header"]              =
+if ( !isset( $wgAutoSitemap["filename"] ) ) { $wgAutoSitemap["filename"] = "sitemap.xml";
```
Comment (dolfinus) [inline AutoSitemap_body.php]: Please don't use spaces after and before brackets `()`.

File: AutoSitemap_body.php [hunk 2]
```diff
@@ -46,207 +46,220 @@
...
+}
+if ( !isset( $wgAutoSitemap["server"] ) ) { $wgAutoSitemap["server"] = isset( $wgCanonicalServer ) ? $wgCanonicalServer : $wgServer;
+}
+if ( !isset( $wgAutoSitemap["notify"] ) ) { $wgAutoSitemap["notify"] = [
+																			'https://www.google.com/webmasters/sitemaps/ping?sitemap=' . $wgAutoSitemap["server"] . $wgScriptPath . '/' . $wgAutoSitemap["filename"],
+																			'https://www.bing.com/webmaster/ping.aspx?sitemap=' . $wgAutoSitemap["server"] . $wgScriptPath . '/' . $wgAutoSitemap["filename"],
+																		];
+}
+
+if ( !isset( $wgAutoSitemap["exclude_namespaces"] ) ) { $wgAutoSitemap["exclude_namespaces"] = [
```
Comment (dolfinus) [inline AutoSitemap_body.php]: Why is so much empty space there?

File: AutoSitemap_body.php [hunk 3]
```diff
@@ -46,207 +46,220 @@
...
 
-if (!isset($wgAutoSitemap["footer"]           )) $wgAutoSitemap["footer"]              = "\n</urlset>";
+if ( !isset( $wgAutoSitemap["footer"] ) ) { $wgAutoSitemap["footer"] = "\n</urlset>";
+}
 
 class AutoSitemap {
 
-    static public function writeSitemap() {
-        global $wgAutoSitemap;
+	public static function writeSitemap() {
```
Comment (dolfinus) [inline AutoSitemap_body.php]: Please don't use tabs.

**Comment by zoranzoki21** [comment]
I think that I should make new pull request, this is so bad.

**Pull request closed without merging**